hooks/build

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker buildx build --platform linux/amd64,linux/arm64,linux/arm/v7 -f $DOCKERFILE_PATH -t $IMAGE_NAME .
```

Comment:
@karaktaka thanks for sharing!

Reply:
That's currently not working if run on docker hub. I don't know exactly why, but I can build it like this locally.